### ISSUE TITLE
Remove HD title screen

### ISSUE THREAD

**Issue by OpenRift412** [sultim-t/prboom-plus-rt#73]
The HD title screen just feels kind of weird and out of place, could we perhaps have it removed or make it optional in the future?

**Comment by bni** [comment]
I agree would be nice to see the classic title screen instead. btw this project is awesome!

**Comment by Rellik66** [comment]
In the meantime, you can remove it yourself by navigating to ovrd\mat\gfx and rename/remove TITLEPIC.ktx2
